### PR TITLE
Adds *insult

### PR DESCRIPTION
**Pull request by Poojawa**
:cl:
add: Added *insult, for when you just can't think of anything more clever
/:cl:

Idea picked up from sawu, refined for the type of class most players have as NPCs. hue.

### CODE REVIEW

**Review by deathride58 — APPROVED** [review]
*beep
*ping
*buzz

You could probably make it extra authentic by making the proc just call `Say()`

**Comment by Poojawa** [comment]
maybe, but, it works as is so w/e

**Comment by Iamgoofball** [comment]
at least credit sawu for the SNPC insult generator you lifted smh

**Comment by Poojawa** [comment]
The only part that's his technically is the basic structure of the json's speech. everything else is mine.

**Comment by Iamgoofball** [comment]
Yeah, but you totally lifted the idea and the entire syntax. Credit the guy.

File: strings/insult.json [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    "curse_words": [
+        "damn",
+        "shit",
+        "arse",
+        "ass",
+		"wank",
+		"jerk",
+		"bitch",
+		"cuck",
+		"ligger"
```
Comment (Wilchenx) [inline strings/insult.json:176]: if ligger is okay add catbeast too

**Pull request merged**